lib/msf/core/modules/external/python/metasploit/__init__.py

```diff
@@ -0,0 +1,5 @@
+import os.path as p
+import sys
+
+DATA_DIR = p.join(p.dirname(p.realpath(__file__)),
```

Comment:
It would be nice if this were less fragile, for example, if certain important variables could be provided by MSF when launching an external module. I'm not suggesting it is resolved in this PR, just a discussion point. cc @acammack-r7

Reply:
I put it in __init__.py so at least it can be defined in a common place

Reply:
I think that eventually we want to have an external config and data storage thing that both the console and the modules would query. As a transitional step to that, I think this is fine for now.

Reply:
Probably the best way to make this less fragile would be if the module just made API calls right into framework directly, rather than also parsing files in the data directory. I'm envisioning external modules eventually even being able to run in docker or vms, where they cannot see metasploit framework's filesystem